modern/src/maki-interpreter/interpreter.js

```diff
@@ -160,7 +160,14 @@ export function* interpret(start, program, stack = []) {
           methodArgs.push(aValue);
         }
         const obj = popStackValue();
-        const ret = obj[methodName](...methodArgs);
+        let ret;
+        if (obj === 0) {
+          // TODO: Check if Winamp throws runtime errors for these
+          console.warn(`Trying to execute ${methodName} on NULL reference`);
+          ret = null;
```

Comment:
I think I'd rather throw here than return null. Could we just throw this message instead of `warn` in it?

I'm guessing the goal here it to get better insight into what's going wrong rather than actually being able to recover.

Reply:
Yea I think it might have had something to do with the  abstraction layer bug 

I'll try it out in real winamp and see if it throws a runtime exception of how it handles it

Reply:
Yea I think it might have had something to do with the  abstraction layer bug 

I'll try it out in real winamp and see if it throws a runtime exception of how it handles it

Reply:
Yea I think it might have had something to do with the  abstraction layer bug 

I'll try it out in real winamp and see if it throws a runtime exception of how it handles it

Reply:
❤️